bioformats2raw/meta.yaml

```diff
@@ -6,18 +6,16 @@ package:
   version: "{{ version }}"
 
 build:
-  number: 0
+  number: 1
 
 source:
   url: https://github.com/glencoesoftware/{{ name }}/releases/download/v{{ version
     }}/{{ name }}-{{ version }}.zip
   sha256: e94e3fd268bd144069d672899ee0b53182d2b3d71519d2fade8bfaedd6c7bf24
 
 requirements:
-  build:
-  - m2-patch    # [win]
   run:
-  - bioformats2raw-libs={{ PKG_VERSION }}-{{ PKG_BUILDNUM }}
+  - bioformats2raw-libs
```

Comment:
The version pin is needed to ensure `conda upgrade bioformats2raw` automatically upgrades the `-libs` package in sync.

Reply:
Fair enough. Building the together, locally, though is no longer working. Any ideas on how to fix that? None of the following worked:

* `-c local`
* `--use-local`
* `conda build biofornats2raw{-libs,}` (i.e. at the same time)
* `... --output-folder X` (i.e. into the same folder)

Reply:
Dunno, maybe you need this?
```
    defaults:
      run:
        shell: bash -el {0}
```
I've no idea why but it's in the README:
https://github.com/conda-incubator/setup-miniconda?tab=readme-ov-file#usage-examples

Reply:
It's failing even locally just with `conda build`.

Reply:
Would the use of `outputs:` be a more modern version of what's going on here?

e.g., https://github.com/AnacondaRecipes/boost-feedstock/blob/master/recipe/meta.yaml